setup.cfg

```diff
@@ -16,7 +16,6 @@ platforms = Linux, Windows, MacOs
 
 
 [options]
-python_requires='>=3'
```

Comment:
Pleas see current version from repository. The problem here are `'`. Without theme everything work fine.